R/loo_compare.R

```diff
@@ -116,7 +116,30 @@ loo_compare.default <- function(x, ...) {
   diffs <- mapply(FUN = elpd_diffs, loos[ord[1]], loos[ord])
   elpd_diff <- apply(diffs, 2, sum)
   se_diff <- apply(diffs, 2, se_elpd_diff)
-  comp <- cbind(elpd_diff = elpd_diff, se_diff = se_diff, comp)
+  # compute probabilities that a model has worse elpd than the best model
+  # using a normal approximation (Sivula et al., 2025)
+  p_worse <- pnorm(0, elpd_diff, se_diff)
```

Comment:
```suggestion
  p_worse <- stats::pnorm(0, elpd_diff, se_diff)
```

will avoid an R cmd check note or warning